reconcile/quay_mirror.py

```diff
@@ -54,15 +54,41 @@ class QuayMirror:
         shard_id=sharding.SHARD_ID,
     )
 
-    def __init__(self, dry_run=False):
+    def __init__(
+        self,
+        dry_run: bool = False,
+        control_file_dir: Optional[str] = None,
+        force_compare_tags: bool = False,
+        compare_tags_interval: int = 86400,
```

Comment:
Aren't defaults provided via click?

Reply:
fixed

Reply:
Just realized reading your comment again, will removing the defaults from Click make this harder to use? Or asked in another way, is Click now complaining if I do not specify the compare_tags_interval?

Reply:
Click doesn't complain but if I don't specify defaults, it will pass `None` explicitly, the defaults in the signature won't be considered and the program will fail. So I needed to put them back again in db05bd38